endpoints/version_test.go

```diff
@@ -0,0 +1,55 @@
+package endpoints
+
+import (
+	"io/ioutil"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestVersionEndpoint(t *testing.T) {
+	var testCases = []struct {
+		description string
+		version     string
+		revision    string
+		expected    string
+	}{
+		{
+			description: "Empty",
+			version:     "",
+			expected:    `{"version":"not-set","revision":"not-set"}`,
```

Comment:
Nitpick: Do you think we should include revision first to mimic the real response order? The test passes fine because JsonEq ignores element order, hence it's just a nitpick.